tests/system/fim/common_tasks/clean_env.yml

```diff
@@ -0,0 +1,167 @@
+---
+  - hosts: all
+    vars_files:
```

Comment:
Include `wazuh_agent.yml`, required variables for the cleaning tasks on Windows.